mgmt_cli_test.py

```diff
@@ -1679,52 +1678,77 @@ def create_backup_and_report(self, mgr_cluster, label: str):
         return task
 
     def run_read_stress_and_report(self, label):
-        stress_queue = []
+        stress_queue = {"read": [], "write": []}
 
-        for command in self.params.get('stress_read_cmd'):
-            stress_queue.append(self.run_stress_thread(command, round_robin=True, stop_test_on_failure=False))
+        for command in self.params.get('stress_cmd'):
+            if " write " in command:
+                stress_queue["write"].append(self.run_stress_thread(command))
+            elif " read " in command:
+                stress_queue["read"].append(self.run_stress_thread(command))
+            else:
+                raise ValueError("Unknown stress command")
+
+        def get_stress_averages(queue):
+            averages = {'op rate': 0.0, 'partition rate': 0.0, 'row rate': 0.0, 'latency 99th percentile': 0.0}
```

Comment:
Generally it's wrong to average latencies.
Consider crating a test named "mixed_latency_during_backup" and using `@latency_calculator_decorator` - we have all these implemented there in a proper way and is sending to Argus all needed stuff out of the box.

Reply:
refactored